docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,241 @@
+Click on the **Services** subtab and click **Create service**.
+Select the `vision` type, then select the `detector:facial-detector` model. Enter a name for your vision service, `"face-detect"` and click **Create**.
+
+Edit the attributes as applicable according to the configuration information on [GitHub](https://github.com/viam-labs/facial-detection):
+
+- `"face_labels"`: Label a photo of the face of each person you want your security system to recognize with the name you want for the label paired with the image path.
+- `"recognition_model"`: The model to use for facial recognition. `"ArcFace"` is chosen as the default for a good balance of speed and accuracy. See [GitHub](https://github.com/viam-labs/facial-detection) for more options.
+- `"detection_framework"`: The detection framework to use for facial detection. `"ssd"` is chosen as the default for a good balance of speed and accuracy.
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "face_labels": {
+    "my_name": "/home/me/my-photo.jpg"
+  },
+  "recognition_model": "ArcFace",
+  "detection_framework": "ssd"
+}
+```
+
+Make sure to name this detector `"face-detect"`.
+
+## Configure a verification system
+
+Now that you've configured your camera, a rough `"people-detect"` detector and a more fine-tuned `"face-detect"` facial detector, you can set up your facial verification system module.
+
+Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/). Click on the **Services** subtab and click **Create service**. Select the `vision` type, then select the `classifier:verification-system` model. Enter a name for your vision service and click **Create**.
+
+To use the facial detector as the verification detector and the people detector as the first two layers of detectors, with **Builder** mode selected copy and paste the following attributes configuration into your service's **Attributes** box:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "trigger_1_confidence": 0.35,
+  "verification_detector": "face-detect",
+  "camera_name": "my-webcam",
+  "trigger_2_confidence": 0.5,
+  "trigger_1_labels": ["Person"],
+  "trigger_2_labels": ["Person"],
+  "disable_alarm": false,
+  "trigger_2_detector": "people-detect",
+  "verification_labels": ["my_name"],
+  "trigger_1_detector": "people-detect",
+  "disarmed_time_s": 10,
+  "countdown_time_s": 10
+}
+```
+
+Name your service `"security"`.
+
+- As the `"people-detect"` detector serves as both trigger detector 1 and 2, set `"trigger_1_labels"` and `"trigger_2_labels"` to `"Person"` to match the bounding box label of `"Person"` you assigned to images in your dataset when training the object detection model behind this detector.
+- As the verification detector is your facial detector `"face-detect"`, set `"verification_labels"` to match each name you assigned to an image in the facial detector modules' `"face_labels"` configuration attribute.
```

Comment:
"As the verification detector is your facial detector" - this presupposes that you've explained what a verification detector is and what it's used for and that it will be the facial detector and I really do not think that is clear at this point.

Reply:
Added more info about identity verification in the facial verification section